packages/cli/commands/executeAll.ts

```diff
@@ -137,7 +154,12 @@ export class ExecuteAll extends Command {
 			if (startNode === undefined) {
 				// If the workflow does not contain a start-node we can not know what
 				// should be executed and with which data to start.
-				GenericHelpers.logOutput(`Workflow ID ${workflowData.id} cannot be started as it does not contain a "Start" node.`);
+				if (json === true) {
+					executionResult.error = 'Workflow cannot be started as it does not contain a "Start" node.';
+					GenericHelpers.logOutput(JSON.stringify(executionResult, null, 2));
```

Comment:
Instead of sending small chunks of data, I think it might be better to append these to a single object so that we have a single output that can be parsed as JSON.

Right now, we have a lot of small JSON objects but if we pass this huge string to a JSON parser, it won't be considered a valid JSON.

Maybe we should wrap it in an object with more data, like number of workflows executed, summary of success and failures and an array of all nodes covered.

Reply:
i wrapped all results in one single object, which contains

- workflowsNumber:number -> number of total execution
- summary:{
      failedExecutions: number -> number of failed executions
      succeeedExecution: number -> number of successful executions
      exceptions: number -> number of exception during workflow execution
    }
- nodesCovered: {
 		  [key:string]:number
     } -> an object contains the covered nodes as keys and their occurrences as value
- executions:[] -> contain the executions results